virttest/virsh.py

```diff
@@ -1588,12 +1587,6 @@ def undefine(name, options=None, **dargs):
     if options is not None:
         cmd += " %s" % options
 
-    # When the platform is aarch64 or x86_64, both seabios and ovmf
-    # guests will use --nvram option
-    if platform.machine() in ["aarch64", "x86_64"]:
-        if options is None or "--nvram" not in options:
-            cmd += " --nvram"
-
     LOG.debug("Undefine VM %s", name)
```

Comment:
I think it can be also removed as well

Reply:
removed

Reply:
After discussion in the team, we decide to keep the log.